app/controllers/schools/placement_dates_controller.rb

```diff
@@ -46,6 +46,22 @@ def update
     end
   end
 
+  def close
+    confirmed = ActiveModel::Type::Boolean.new.cast(
+      params["schools_placement_dates_close_confirmation_form"]["confirmed"]
+    )
+
+    Bookings::PlacementDate.find(params[:placement_date_id]).update!(active: false) if confirmed
```

Comment:
Do these requests need to be scoped to the current school? otherwise what stops me putting in a random school ID into the query params and closing their placement date?

Reply:
Done 👍